kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/filter/authorization/AuthzIT.java

```diff
@@ -356,10 +357,34 @@ protected static JsonNode maybeClobberedUuid(JsonNode uuid) {
         return uuid;
     }
 
+    protected static JsonNode maybeClobberedString(JsonNode uuid) {
+        if (uuid != null && uuid.isTextual()) {
+            return TextNode.valueOf("CLOBBERED");
+        }
+        return uuid;
+    }
+
     protected static void clobberUuid(ObjectNode root, String propertyName) {
         root.replace(propertyName, maybeClobberedUuid(root.get(propertyName)));
     }
 
+    protected static void clobberString(ObjectNode root, String propertyName) {
+        root.replace(propertyName, maybeClobberedString(root.get(propertyName)));
+    }
+
+    protected static void clobberInt(ObjectNode root, String propertyName, int replacement) {
+        root.replace(propertyName, maybeClobberedInt(root.get(propertyName), replacement));
+    }
+
+    private static JsonNode maybeClobberedInt(JsonNode jsonNode, int replacement) {
+        if (jsonNode != null && jsonNode.isNumber()) {
+            return new IntNode(replacement);
```

Comment:
I guess we should prefer to replace with a "CLOBBERED" String, there's no reason to keep the node type the same. It will make it more visible in test failures.